README.md

```diff
@@ -1 +1,66 @@
-# library-js
+# Library-js
+
+> Libary-js is a simple introductory javascript project to demonstrate how to create a form using HTML and use Javascript to populate it.
+
+
+## Built With
+
+- JavaScript
+- HTML
+- CSS
+
+
+### Prerequisites
+    
+    A computer running Windows, Linux or MacOS
+
+### Setup
+
+    # Windows
+
+    Download and run Node and NPM @ https://nodejs.org/en/download/
+
+    # Mac/Linux
+
+    Open Terminal and type: nvm install node
+
+
+### Run
+
+    Copy all the project files to a folder.
+
+    Open Terminal and change directory to the project folder
+    
+    Open index.html on your browser
+
```

Comment:
## Improve on the setup instructions. ✍️ 

The setup instructions are not really clear as to what the user of the repo should do. While instructions to download node is okay, having it as a prerequisite would be much better. In place/addition to these instructions let's work on the following: 👇 

- [ ] Add instructions on how to best clone and run the repo locally. This will ensure that the project has a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))